wp-modules/app/js/src/utils/test/toKebabCase.ts

```diff
@@ -1,6 +1,6 @@
-import convertToSlug from '../convertToSlug';
+import toKebabCase from '../toKebabCase';
 
-describe( 'convertToSlug', () => {
+describe( 'toKebabCase', () => {
```

Comment:
Just a name change, as `convertToSlug()` isn't for a pattern slug, just for kebab-case.

Reply:
Thanks!